packages/cashier/src/components/crypto-transactions-history/crypto-transactions-history.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { DataList, Icon, Loading, MobileWrapper, Table, Text } from '@deriv/components';
+import { DataList, Icon, Loading, MobileWrapper, Table, Text, Modal } from '@deriv/components';
```

Comment:
@aum-deriv pls always try to use alphabetic order in import, props, etc. for better readability 
```suggestion
import { DataList, Icon, Loading, MobileWrapper, Modal, Table, Text} from '@deriv/components';
```